Python/thread.c

```diff
@@ -139,14 +139,46 @@ happen to be PyObject*, these functions don't do refcount operations on
 them either.
 
 The GIL does not need to be held when calling these functions; they supply
-their own locking.  This isn't true of PyThread_create_key(), though (see
+their own locking.  This isn't true of PyThread_tss_create(), though (see
 next paragraph).
 
-There's a hidden assumption that PyThread_create_key() will be called before
+There's a hidden assumption that PyThread_tss_create() will be called before
 any of the other functions are called.  There's also a hidden assumption
-that calls to PyThread_create_key() are serialized externally.
+that calls to PyThread_tss_create() are serialized externally.
 ------------------------------------------------------------------------ */
 
+/* Thread Specific Storage (TSS) API
+
+   Implementation part of common
```

Comment:
Suggested rephrasing for this comment: "Cross-platform components of TSS API implementation"